docs/rpi3.md

```diff
@@ -206,44 +206,42 @@ $ sudo rm -rf /srv/nfs/rpi/boot/*
 ```
 
 ### 5.4 Update uboot.env
-We need to make a couple of changes to that file to ensure that it will try to
-boot using everything we have prepared. So, start by inserting the UART cable
-and open up `/dev/ttyUSB0`
-```bash
+Actually, there are two ways to update uboot.env. First, you can edit
+`build/rpi3/firmware/uboot.env.txt` file, which is used as simple text source for
+generation of uboot.env during build. And you can just edit u-boot env via UART
+and save new values to uboot.env. By using the second way you can avoid rebuilding
+and copying uboot.env to SD card, but on the other hand, it's easier to break
+something while performing "hot" edit.
+
+#### 5.4.1 Edit uboot.env.txt
+All you need to do is to edit network configuration in `build/rpi3/firmware/uboot.env.txt`.
+You have to change value of `serverip` to the IP address of your NFS/TFTP server,
+`gatewayip` to your router IP address and `nfspath` to the exported path, where root FS
+is stored (`/srv/nfs/rpi`). Then you need to generate new `uboot.env`:
+```
```

Comment:
I've recently started to add "bash" to code sections to get syntax highlight etc.
```
 "```bash"
```